### DISCUSSION TITLE
ATTACH with KEY

### DISCUSSION THREAD

**Discussion by pawelsalawa** [utelle/SQLite3MultipleCiphers#25] [General]
I was looking in documentation, whether the SQLite3 MC still supports the ATTACH with KEY clause. I found it mentioned in architecture & patching section. I think it may be worth to mention it also in Cipher configuration -> SQL Interface. At least this was the first place I looked for it.

**Comment by utelle** [comment]
You are right, this feature of the `ATTACH` SQL command should be mentioned more prominently.
